Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.2
+import PackageDescription
+
+let package = Package(
+    name: "SendGridKit",
```

Comment:
Everything looks good to me. Can we update the package name to be `sendgrid-kit`. This falls in line with `stripe-kit`, `google-cloud-kit` as well as `async-http-client`.